ci/azure/linux_script

```diff
@@ -69,6 +69,8 @@ release/bin/zig build test -Denable-qemu -Denable-wasmtime
 
 # Look for HTML errors.
 tidy -qe ../zig-cache/langref.html
+# look for formatting errorrs
+release/bin/zig fmt --check ../lib/std ../src ../tools ../test ../build.zig
```

Comment:
Wouldn't it make sense to do this before running the tests so people don't have to wait ~3 hours for the test to runs before seeing that their code isn't properly formatted?

Reply:
I thought the opposite - you want formatting errors to be the last thing to run since it is easily fixable, but the tests are not. Now that I think about it, it would make more sense to have it before.

Reply:
I see your point, I think ideally the fmt check would happen in a different job and allow the actual testing to proceed in parallel even if errors are found.

Reply:
I only enabled it in azure, so the other tests should run. I don't think a fully other job just for fmt is good because it will also have to bootstrap zig which takes cpu time.